src/alfasim_sdk/_internal/version.py

```diff
@@ -1 +1 @@
-__version__ = "0.20.0"
+__version__ = "0.21.0"
```

Comment:
Isn't this good? I mean we did forget to update this version in the past... if it is marked as `.dev` it will be clear it needs updating.

But I will defer the decision to you then.

Reply:
I will consider this when tackling ASIM-5648.